### ISSUE TITLE
Question about domain's partition?

### ISSUE THREAD

**Issue by machanic** [HAHA-DL/MLDG#3]
After I read the paper, one question remains for me, you wrote that the domain S is splited into meta-train domains and meta-test domains.
For example, the domain can be the style of images, image A has two styles:  cartoon and realistic image.
Then in the Alg. 1, the meta train loss is calculated on the A's cartoon style, and meta val loss is calculated on the A's realistic style?
Or the two loss are computed on completed different images' styles: A's cartoon style for meta train loss, and image B's realistic style for meta val loss?
yours sincerely.
Thank you!
